src/check.py

```diff
@@ -1,3 +1,26 @@
 """
 This file includes stuff actual checking
-"""
\ No newline at end of file
+"""
```

Comment:
Change this doc comment with something that makes sense or remove it. It was put as just a placeholder.